ui-tests/src/test/java/io/syndesis/qe/steps/other/DocsVerificationSteps.java

```diff
@@ -0,0 +1,46 @@
+package io.syndesis.qe.steps.other;
+
+import static org.junit.Assert.assertEquals;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.Selenide;
+
+import java.util.HashMap;
+
+import cucumber.api.java.en.Then;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class DocsVerificationSteps {
+    private HashMap<String, String> versions = new HashMap<>();
+    private static String versionOnAboutPage;
+
+    public DocsVerificationSteps() {
+        versions.put("1.9-SNAPSHOT", "7.5");
+    }
+
+    @Then("^check version in about page$")
+    public void checkVersion() {
+        final By SYNDESIS_VERSION = By.cssSelector("[data-testid=\"about-modal-content-version-list-item\"]");
+        assertThat($(SYNDESIS_VERSION).shouldBe(visible).getText())
+            .isNotEmpty();
+        versionOnAboutPage = $(SYNDESIS_VERSION).getText();
```

Comment:
not needed variable, you can use `$(SYNDESIS_VERSION).getText()` in equals directly